app/models/katello/subscription_status.rb

```diff
@@ -43,7 +43,13 @@ def to_global(_options = {})
 
     def to_status(options = {})
       return UNKNOWN unless host.subscription_facet.try(:uuid)
-      status_override = 'unsubscribed_hypervisor' if host.subscription_facet.hypervisor && !host.subscription_facet.candlepin_consumer.entitlements?
+      if host.subscription_facet.hypervisor
+        if host.organization.simple_content_access?
+          status_override = 'simple_content_access'
+        elsif !host.subscription_facet.candlepin_consumer.entitlements?
+          status_override = 'unsubscribed_hypervisor'
+        end
+      end
       status_override ||= options.fetch(:status_override, nil)
       status = status_override || Katello::Candlepin::Consumer.new(host.subscription_facet.uuid, host.organization.label).entitlement_status
```

Comment:
Extra credit: The `unsubscribed_hypervisor?` check may have already initialized a Katello::Candlepin::Consumer object. See subscription_facet.rb#candlepin_consumer So, we can use it again here:

```
status = status_override || host.subscription_facet.candlepin_consumer.entitlement_status
```

Reply:
Updated to this, thank you for the pointer